packages/lab/src/calendar/useCalendar.ts

```diff
@@ -30,6 +29,8 @@ interface BaseUseCalendarProps {
     visibleMonth: DateValue
   ) => void;
   isDayUnselectable?: (date: DateValue) => UnselectableInfo | boolean | void;
```

Comment:
I think I mentioned this previously but not sure what change was made. Should this be `false` rather than `boolean` to force users to provide a value or void for false